packages/@aws-cdk/aws-lambda/lib/image-function.ts

```diff
@@ -41,8 +43,14 @@ export abstract class DockerImageCode {
    */
   public static fromImageAsset(directory: string, props: AssetImageCodeProps = {}): DockerImageCode {
     return {
-      _bind() {
-        return new AssetImageCode(directory, props);
+      _bind(architecture?: Architecture) {
+        return new AssetImageCode(directory, {
+          /**
+           * determine the platform from `architecture`.
+           */
```

Comment:
I change to this way because of this:
![image](https://user-images.githubusercontent.com/91827012/182531388-143f2ec2-1d42-487b-94c5-7cf0d09ef51f.png)

Updated.